src/System.Private.ServiceModel/tests/Stress/SharedPoolsOfWCFObjects/CallStats.cs

```diff
@@ -8,7 +8,7 @@
 using System.Threading;
 using System.Diagnostics;
 
-namespace WCFClientStressTests
+namespace SharedPoolsOfWCFObjects
```

Comment:
NIT: just curious why we renamed; we have a fairly consistent namespace naming scheme, so if it were to be renamed, you could consider using that. We can consider this after merge though.